### ISSUE TITLE
[1.16.5/Gunpowder 0.5.7] Server load stopped

### ISSUE THREAD

**Issue by JuergenSchT** [Gunpowder-MC/Gunpowder#122]
**Information**
- Minecraft Version: 1.16.5
- Fabric API Version: 0.40.6+1.16
- OS: Ubuntu 20.04.1 LTS
- Other installed mods: FAPI, ,FabricLang Kotlin [1.5.31]

**Describe the issue**
Server load stops after spewing a bunch of lines on the blog. No crash log is generated.
Gunpowder 0.5.3 - 0.5.6 also cause this error (0.3.6 - 0.5.2 fail with a different error).

**Crash/Error logs**
[[LINK](https://gist.github.com/JuergenSchT/26ffaa9c6b85851113c356c149c76fee)]

**To Reproduce**
Start a server with the aforementioned mods.

**Expected behavior**
Expected the server to fully load.

**Additional context**
Gunpowder Base 0.2.10 works normally on the same droplet, on the mainline server (which runs on 1.16.4).

**Comment by martmists-gh** [comment]
Can you send your gunpowder.yaml file?

**Comment by martmists-gh** [comment]
It might be that your gunpowder.db comes from an old version, try regenerating it and filling it in again

**Comment by JuergenSchT** [comment]
tl;dr it appears the matter is solved
-----------------------------------------------------------------------------------
> Can you send your gunpowder.yaml file?

Sorry I took so long.
The log above comes from a fresh server instance, where every file was freshly created, _except_ for the config folder (which was copied from the "update-in-progress" instance. Therefore, gunpowder.yaml looks like this:
```
# Main Gunpowder Config

# Database-related stuff
database:
  # Available modes: sqlite, postgres, mysql
  mode: sqlite

  # Host, unused for sqlite
  host: localhost

  # Port, unused for sqlite
  # Default MySQL: 3306 | Default Postgres: 5432
  port: 3306

  # DB credentials, unused for sqlite
  # Make sure to change these!
  username: JuergenSchT
  password: [REDACTED]
  ```
gunpowder.db was not found on this testing folder, apparently it was never created due to the server not loading properly.
Just now, I have tried deleting gunpowder.yaml, and the testing server finally loads. It also works on the "update-in-progress" server.
I noticed that these lines were added to the config in the most recent versions of Gunpowder for 1.16, which were not there in 0.2.10 and the like:
```
  # Name of the database
  database: gunpowder
```
These lines were not added automatically to the config. Without them, the server did not load. After adding them manually, the server loaded as expected, without touching any existing gunpowder.db. There doesn't seem to be anything usual after that.

**Comment by martmists-gh** [comment]
Yeah, back in 0.3.0 I added to the changelogs a notice that upgrading would break because of that, glad it works now!
